autoexpect/src/leiningen/autoexpect.clj

```diff
@@ -22,8 +22,9 @@
   Reports results to growl and STDOUT."
   [project & args]
   (let [should-growl (some #{:growl ":growl" "growl"} args)
-        should-notify (some #{:notify ":notify" "notify"} args)]
+        should-notify (some #{:notify ":notify" "notify"} args)
+        change-only (some #{:notify ":change-only" "change-only"} args)]
```

Comment:
You're right, nice catch! Was a little quick on the copy-paste, and of course it worked fine when testing.